libimage/normalize_test.go

```diff
@@ -83,7 +84,7 @@ func TestNormalizePlatform(t *testing.T) {
 			platform{"linux", "arm", "v6"},
 		},
 	} {
-		os, arch, variant := NormalizePlatform(test.input.os, test.input.arch, test.input.variant)
+		os, arch, variant := lplatform.NormalizePlatform(test.input.os, test.input.arch, test.input.variant)
```

Comment:
Can this be renamed to platform.Normalize()

Reply:
There is a struct called `platform` defined for these tests so when I try to do `platform.` it picks the struct instead of the library. I can rename the struct in the test but that is much more changes than needed, so chose to use a diff name for the platform library.

Reply:
Can this be renamed to lplatform.Normalize()